geode-docs/developing/geode_apis_compatible_with_redis.html.md.erb

```diff
@@ -27,7 +27,7 @@ optional password authentication.
 
 <img src="../images_svg/geode_apis_compatible_with_redis.svg" class="image" />
 
-## <a id="using-the-api" class="no-quick-link"></a>Using the <%=vars.product_name%> API compatible with Redis
+## <a id="using-the-api" class="no-quick-link"></a>Using the <%=vars.product_name%> APIs Compatible with Redis
```

Comment:
Doesn't look like this got changed.  We talked in previous comments about making it `Using the Geode APIs Compatible with Redis`